src/vmecpp/cpp/vmecpp/simsopt_compat/_fortran_wout_adapter.py

```diff
@@ -435,15 +435,16 @@ def from_vmecpp_wout(vmecpp_wout: _VmecppWOutLike) -> FortranWOutAdapter:
 
         return FortranWOutAdapter(**attrs)
 
-    def save(self, out_path: Path) -> None:
+    def save(self, out_path: str | Path) -> None:
         """Save contents in NetCDF3 format.
 
         This is the format used by Fortran VMEC implementations and the one expected by
         SIMSOPT.
         """
+        _out_path = Path(out_path)
```

Comment:
very minor, but it's more typical in Python to just:
```suggestion
        out_path = Path(out_path)
```

which has the advantage that you can't get confused in the body between out_path and _out_path

Reply:
For a second I forgot that Python was pass by value... Ofc I'll change that